### DISCUSSION TITLE
Any interest in hardening against screenshot/screencapture in OSX?

### DISCUSSION THREAD

**Discussion by canselcik** [bitwarden/clients#10689] [Password Manager]
### Code Contribution Proposal

If there is any interest, I try to put together a PoC. 

https://stackoverflow.com/questions/65820645/prevent-screencapture-from-capturing-app-screen-on-macosx
https://github.com/heardrwt/RHAdditions/blob/master/RHAdditions/NSWindow%2BRHPreventCaptureAdditions.m

**Comment by djsmith85** [comment]
@canselcik Thank you for your raising this idea and offering your help.

It seems @quexten beat you too it, with following the [related thread on reddit](https://github.com/bitwarden/clients/pull/10707).

Here is his Pull Request: https://github.com/bitwarden/clients/pull/10707

I'm going to close this conversation, but feel free to comment on the PR if you have any further input.

Closing this now.
